geps/gep-1897/index.md

```diff
@@ -281,6 +271,23 @@ reverse proxy. This is shown as **bolded** additions in step 6 below.
 6. Lastly, the reverse proxy **optionally performs a TLS handshake** and forwards the request to one or more objects,
 i.e. Service, in the cluster based on backendRefs rules of the HTTPRoute **and the TargetRefs of the BackendTLSPolicy**.
 
+## Future plans
+
+In order to scope this GEP, some some changes are deferred to a near-future GEP.
+This GEP intends to add the ability for additional control by gateway clients to override TLS settings, following previously established patterns of [consumer and producer policies]([glossary](https://gateway-api.sigs.k8s.io/concepts/glossary/?h=gloss#producer-route)).
+Additionally, more contextual control over when to apply the policies will be explored, to enable use cases like "apply TLS only from this route" ([issue](https://github.com/kubernetes-sigs/gateway-api/issues/3856)).
+
+While the details of these plans are out of scope for this GEP it is important to be aware of the future plans for the API to ensure the immediate-term plans are future-proofed against the proposed plans.
+
+Implementations should plan for the existence of future fields that may be added that will control where the TLS policy applies.
+These may include, but are not limited to:
+
+* `spec.targetRefs.namespace`
+* `spec.targetRefs.from`
+* `spec.mode`
```

Comment:
Don't forget that we already have [backendTLS](https://gateway-api.sigs.k8s.io/reference/spec/#gatewaybackendtls) that configures TLS settings for when this Gateway is connecting to backends with TLS.  It overrides BackendTLSPolicy.

Reply:
I am aware of that field, thanks!